src/extension.ts

```diff
@@ -195,6 +196,7 @@ export async function activate(context: vscode.ExtensionContext) {
 	)
 
 	registerAutocomplete(context) // kilocode_change
+	registerGhost(context) // kilocode_change
```

Comment:
Should we hide this behind a dev flag for a week or so while we iterate on it? Maybe something more hidden than an experimental flag?